src/service/receipt.service.ts

```diff
@@ -0,0 +1,130 @@
+
+import chalk from 'chalk';
+import {
+    ArtifactExpiryReceipt,
+    BalanceChangeReceipt,
+    BalanceTransferReceipt,
+    InflationReceipt,
+} from 'nem2-sdk';
+
+export class ReceiptService {
+    constructor() {
+
+    }
+
+    /**
+     * @description The logic of formatting Receipt info
+     * @param receipt
+     * @returns {string}
+     */
+    public formatReceiptToFilter(statement: any): string {
```

Comment:
As it is  a quite long function, I'd suggest to split it into at least two parts:

* formatTransactionStatements
* formatResolutionStatements: can mix both address and mosaic resolution statements logic as most of the fields are the same.